app/views/notifications.scala.html

```diff
@@ -28,21 +31,21 @@
         <table>
             <tr class="table-row table-head">
                 <th class="table-cell" id="reference">@messages("notifications.reference")</th>
-                <th class="table-cell" id="action">@messages("notifications.action")</th>
-                <th class="table-cell" id="dateOfRequest">@messages("notifications.dateOfRequest")</th>
                 <th class="table-cell" id="dateOfResponse">@messages("notifications.dateOfResponse")</th>
                 <th class="table-cell" id="roe">@messages("notifications.roe")</th>
                 <th class="table-cell" id="soe">@messages("notifications.soe")</th>
             </tr>
 
             @for(notification <- notifications){
                 <tr class="table-row">
-                    <td class="table-cell">@notification.conversationId</td>
-                    <td class="table-cell">Action</td>
-                    <td class="table-cell">Date of Request</td>
-                    <td class="table-cell">Date of Response</td>
-                    <td class="table-cell">ROE</td>
-                    <td class="table-cell">SOE</td>
+                    <td class="table-cell">@Html(notification.ucrBlocks.map(_.ucr).mkString("<br>"))</td>
```

Comment:
Do you need this `@Html`? What situation cover this? What `mkString` change?

Reply:
They're both to allow printing every value in a separate row inside a single table cell.

Reply:
This is quite fast way to get XSS. Are we able to write little partial for working with seqences ?